images/win/Windows2016-Readme.md

```diff
@@ -40,6 +40,7 @@
 
 ### Tools
 - 7zip 19.00
+- aria2 1.35.0
```

Comment:
There is no need to update the Readme manually, need to update Software Report Generator instead
https://github.com/actions/virtual-environments/tree/main/images/win/scripts/SoftwareReport

Reply:
Done in 4543939c7c0c27865a0f164306aee086320565ae